cog/daily_charge.py

```
@@ -23,95 +26,242 @@
+        last_charge = user_data.last_charge
+        downtime_list = get_downtime_list()
+
+        return any(
+            downtime.start.date() - timedelta(days=1)
+            <= last_charge.date()
+            <= downtime.start.date()
+            for downtime in downtime_list
+        )
+
+    # TODO: inherit a MySQLCursorAbstract to add method about these or consider to add self.cursor
+    def reward(
```

Ruff (0.8.2)

Missing return type annotation for public function `reward`

Add return type annotation: `None`

(ANN201)